README.md

```diff
@@ -679,6 +679,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [Ubiq Security](https://ubiqsecurity.com/) — Encrypt and decrypt data with three lines of code and automatic key management. Free for one application and up to 1,000,000 encryptions per month.
   * [Virgil Security](https://virgilsecurity.com/) — Tools and services for implementing end-to-end encryption, database protection, IoT security, and more in your digital solution. Free for applications with up to 250 users.
   * [Virushee](https://virushee.com/) — Privacy-oriented file/data scanning powered by hybrid heuristic and AI-assisted engine. It is possible to use internal dynamic sandbox analysis. Limited to 50MB per file upload
+  * [Vulert](https://vulert.com) - Vulert continuously monitors your open-source dependencies for new vulnerabilities, recommends fixes, without requiring installation or access to your codebase. Free for open-source projects.
```

Comment:
100 users does not “solidified its place as a leading software composition analysis tool”

:)

Reply:
@ripienaar thanka for comment, these are just top 100 companies we have more than 10K active users using vulert.